doc/source/data/key-concepts.rst

```diff
@@ -36,75 +37,72 @@ This flexibility is a unique characteristic of Ray Datasets.
 Reading Data
 ============
 
-Datasets uses Ray tasks to read data from remote storage. When reading from a file-based datasource (e.g., S3, GCS), it creates a number of read tasks proportional to the number of CPUs in the cluster. Each read task reads its assigned files and produces an output block:
+Datasets uses Ray tasks to read data from remote storage. Each read task reads its assigned files and produces an output block:
 
 .. image:: images/dataset-read.svg
    :align: center
 
 ..
   https://docs.google.com/drawings/d/15B4TB8b5xN15Q9S8-s0MjW6iIvo_PrH7JtV1fL123pU/edit
 
-The parallelism can also be manually specified, but the final parallelism for a read is always capped by the number of files in the underlying dataset. See the :ref:`Creating Datasets Guide <creating_datasets>` for an in-depth guide
-on creating datasets.
+You can manually specify the number of read tasks, but the final parallelism is always capped by the number of files in the underlying dataset.
+
+For an in-depth guide on creating datasets, read :ref:`Creating Datasets <creating_datasets>`.
 
 Transforming Data
 =================
 
-Datasets can use either Ray tasks or Ray actors to transform datasets. By default, tasks are used. Actors can be specified using ``compute=ActorPoolStrategy()``, which creates an autoscaling pool of Ray actors to process transformations. Using actors allows for expensive state initialization (e.g., for GPU-based tasks) to be cached:
+Datasets use either Ray tasks or Ray actors to transform datasets. By default, Datasets uses tasks.
```

Comment:
"Datasets" is singular, and Datasets uses task/actors to transform data/blocks, not datasets.
```suggestion
Datasets uses either Ray tasks or Ray actors to transform data blocks. By default, Datasets uses tasks.
```